networking_generic_switch/devices/corsa_devices/__init__.py

```diff
@@ -169,6 +178,14 @@ def del_network(self, segmentation_id):
         sw_ip_addr = self.config['switchIP']
         url_switch = protocol + sw_ip_addr
 
+        isVFCHost = True
+        if not 'VFCHost' in self.config or not self.config['VFCHost'] == 'True':
+            LOG.info("PRUTH: Skipping VFC Deletion: isVFCHost " + str(isVFCHost))
+            isVFCHost = False
```

Comment:
Same comment as above; I don't think this var is necessary.

Reply:
Same comment about static provisioning.